contracts/infrastructure/WalletFactory.sol

```diff
@@ -218,8 +219,33 @@ contract WalletFactory is Owned, Managed {
      */
     function validateInputs(address _owner, address _versionManager, address _guardian, uint256 _version) internal view {
         require(_owner != address(0), "WF: owner cannot be null");
-        require(IModuleRegistry(moduleRegistry).isRegisteredModule(_versionManager), "WF: invalid _versionManager");
```

Comment:
We accept the fact that wallets could be created with a non-Argent implementation of `IVersionManager` do we?

Reply:
I guess we need to remove this check otherwise we have a way to censure the creation of a wallet (by deregistering the target `VersionManager` from the `ModuleRegistry`). And it remains secure for the user since we have a signature from Argent on the wallet address (which includes the target versionManager address in the hash).

In that case the `ModuleRegistry` is no longer needed in the factory and can be removed entirely from this contract (constructor and `changeModuleRegistry` method).

Reply:
Also, the method can now be marked as `pure`.